### ISSUE TITLE
Allow comments in weird places?

### ISSUE THREAD

**Issue by shaunix** [projectmallard/projectmallard.org#11]
A very common validation error is that people have comment elements intermingled with sections, or after all section elements. They use comment either to remove some section or to make notes about what sections to add in a particular place.

Comments can't go in these places now, because they're just regular block elements. We don't want to allow any block content in these places. But we could special-case comments.

Relatedly: should comment elements intermixed with sections be able to hold a section child instead of blocks? Kind of crazy, but useful, and I've seen people try to do it.

**Comment by kittykat** [comment]
+1

**Comment by shaunix** [comment]
Yes to comments between sections and after the last section. No to sections within comments.

**Comment by shaunix** [comment]
Bumping to Mallard 1.2
